<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="extentia.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Erro Validations Test(failed)">
    <classes>
      <class name="extentia.tests.ErrorValidationsTest">
        <methods>
          <include name="launchApplication"/>
          <include name="productErrorValidation"/>
          <include name="tearDown"/>
          <include name="loginErrorValidation"/>
        </methods>
      </class> <!-- extentia.tests.ErrorValidationsTest -->
    </classes>
  </test> <!-- Erro Validations Test(failed) -->
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)">
    <classes>
      <class name="extentia.tests.SubmitOrderTest">
        <methods>
          <include name="orderHistoryTest"/>
          <include name="launchApplication"/>
          <include name="submitOrder" invocation-numbers="1"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- extentia.tests.SubmitOrderTest -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
